pkgs/applications/misc/oneko/default.nix

```diff
@@ -1,13 +1,13 @@
 { lib, stdenv, fetchFromGitHub, imake, gccmakedep, xlibsWrapper }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (final: {
```

Comment:
https://github.com/NixOS/nixpkgs/pull/119942/files#diff-c032817f6a62e23767f4c7706e35eef36e8353aa5ba5d6f9bce8c6a8cc0db01dR53

` If only a one-argument function is written, the argument has the meaning of `previousAttrs`.`

you should use `finalAttrs: previousAttrs:`